ci-operator/step-registry/ipi/install/vsphere/registry/ipi-install-vsphere-registry-ref.yaml

```diff
@@ -8,7 +8,7 @@ ref:
       memory: 100Mi
   env:
   - name: SETUP_IMAGE_REGISTRY_WITH_PVC
-    default: "false"
+    default: "true"
```

Comment:
If this behavior become default, then we can remove cucushift/installer/rehearse/vsphere/ipi/disconnected/image-registry-pvc/ and update related ci jobs.

Reply:
Does it have to become default? I don't know on other platforms whether we already have default storageclasses, especially the "none" platform

Reply:
Checked the affected [jobs](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pj-rehearse/openshift/release/43694/6da7f58689b4539728bc5727fa8b6f928d26dffb), there are no "none" profiles.